plugin/build.gradle

```
@@ -31,7 +31,7 @@ java {
   targetCompatibility JavaVersion.VERSION_11
 }
 
-group = 'com.jraska.module.graph.assertion'
+group = 'com.github.jokoolle-183.modules-graph-assert'
```

```suggestion
group = 'com.jraska.module.graph.assertion'
```

I guess the change was an accident? :)

---

Yup, reverting it